dotnet/tests/MicroFocus.CafApi.QueryBuilder.Vqll.Builder.SystemJson.Tests/SystemJsonFilterWriterTest.cs

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2022 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Xunit;
+
+using System.Text;
+using System.Text.Json;
+
+namespace MicroFocus.CafApi.QueryBuilder.Vqll.Builders.SystemJson.Tests
+{
+    public sealed class SystemJsonFilterWriterTest
+    {
+        [Fact]
+        public void TestTransformBetweeenEnd()
+        {
+            Filter<string> clause = FilterFactory.Between("REPOSITORY_ID", null, 2345L);
+            VerifyVqllTransformation(clause, @"[""between"",""REPOSITORY_ID"",null,2345]");
+        }
+
+        private static void VerifyVqllTransformation(Filter<string> clause, string expectedTransformation)
+        {
+            string clauseInSqVqll = ToVqll(clause);
+            Assert.Equal(clauseInSqVqll, expectedTransformation);
+        }
+
+        private static string ToVqll(Filter<string> filter)
+        {
+            using var stream = new MemoryStream();
+            using var jsonWriter = new Utf8JsonWriter(stream);
+            SystemJsonFilterWriter.WriteToJsonArray(filter, jsonWriter);
```

Comment:
Making it an extension method means it can be called like this, just as if it was an instance method:

```suggestion
            filter.WriteToJsonArray(jsonWriter);
```